extensions/cornerstone/src/commandsModule.ts

```diff
@@ -145,6 +145,61 @@ function commandsModule({
   }
 
   const actions = {
+    jumpToMeasurementViewport: ({ annotationUID, measurement }) => {
+      cornerstoneTools.annotation.selection.setAnnotationSelected(annotationUID, true);
+      const { metadata } = measurement;
+
+      const activeViewportId = viewportGridService.getActiveViewportId();
+      const viewportId = cornerstoneViewportService.getViewportIdToJump(activeViewportId, metadata);
```

Comment:
this is a little bit confusing api naming too, not your fault 

```
const viewportId = cornerstoneViewportService.getViewportIdToJump(activeViewportId, metadata);
```

I mean it accepts a viewportId and returns a viewportId? how we can improve it?